aegis_control/launch/ur_driver.launch.py

```diff
@@ -0,0 +1,199 @@
+from launch_ros.actions import Node
+from launch_ros.parameter_descriptions import ParameterFile
+from launch_ros.substitutions import FindPackageShare
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.conditions import IfCondition, UnlessCondition
+from launch.substitutions import (
+    LaunchConfiguration,
+    PathJoinSubstitution,
+)
+
+
+class URConfig:
+    def __init__(self):
+        # TODO: Define common parameters for this UR launch file & UR's URDF in an external YAML file
+        self.ur_type = "ur5e"
+        self.robot_ip = "aegis_ur"
+
+        self.use_tool_communication = "false"
+        self.tool_device_name = "/tmp/ttyUR"
+
+        self.controller_spawner_timeout = "10"  # s
+        self.initial_joint_controller = "scaled_joint_trajectory_controller"
+
+        self.ur_controllers_cfg = PathJoinSubstitution(
+            [FindPackageShare("aegis_description"), "config", "controllers.yaml"]
+        )
+        self.update_rate_config_file = PathJoinSubstitution(
+            [
+                FindPackageShare("aegis_description"),
+                "config",
+                "ur5e",
+                "update_rate.yaml",
+            ]
+        )
+
+
+def generate_launch_description() -> LaunchDescription:
+
+    mock_hardware = LaunchConfiguration("mock_hardware")
+    # It is necessary to properly parse ur_controllers_cfg YAML file
+    tf_prefix = LaunchConfiguration("tf_prefix")
```

Comment:
Make this comment more explicit, that the `tf_prefix` arg is necessary for parsing the `ur_robot_driver`'s _controllers.yaml_ configuration.